src/components/query/QuerySummary.vue

```diff
@@ -117,15 +117,21 @@
         </dd>
       </dl>
     </el-row>
+    <el-row v-if="query.run.warnings && query.run.warnings.length > 0">
+      <el-collapse>
+        <run-warnings :warnings="query.run.warnings"></run-warnings>
+      </el-collapse>
+    </el-row>
```

Comment:
So it means that the query will show all run warnings event if not related to this specific query?

Reply:
It's not possible currently to differentiate wether a warning impact your query or not. On the other hand, if you have warnings on your input files, high chances that your query is not doing what it is suppose to do.. So I prefer displaying them all.

Reply:
Ok so we need to make it clear for the user that the warnings are about the input file

Reply:
Changed in 1f3212d and 44a7af1